docs/hello_nextflow/01_hello_world.md

```diff
@@ -320,25 +320,25 @@ It allows us to retrieve the desired output file without having to dig down into
 
 In the workflow script file `hello-world.nf`, make the following code modification:
 
-_Before:_
+=== "After"
 
-```groovy title="hello-world.nf" linenums="6"
-process sayHello {
+    ```groovy title="hello-world.nf" linenums="6" hl_lines="2 3 4"
+    process sayHello {
 
-    output:
-        path 'output.txt'
-```
+        publishDir 'results', mode: 'copy'
 
-_After:_
+        output:
+            path 'output.txt'
+    ```
 
-```groovy title="hello-world.nf" linenums="6"
-process sayHello {
+=== "Before"
 
-    publishDir 'results', mode: 'copy'
+    ```groovy title="hello-world.nf" linenums="6" hl_lines="2"
```

Comment:
I don't think that we should highlight anything in the "Before" block. Unless maybe it's being deleted, or changed? I find the whitespace highlight here confusing anyway 😅

Reply:
I believe there are situations where it would help to have highlights in the before block, but I'm fine removing the highlights until we run into such situations.

Reply:
What do you think of 4.1.2 in Part 2?